### PR TITLE
[New Fundamental] Resilient cloud architecture - 200 level overview

### PR DESCRIPTION
**Pull request by setheliot**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
@setheliot , do you have a reviewer for this one lined up?

**Comment by setheliot** [comment]
@cobusbernard yes, it has already gone through tech review with multiple SMEs.  It is currently blocked on another issue that I will share via DM

File: concepts/resilience-essentials/index.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
+---
```
Comment (DalM2023) [inline concepts/resilience-essentials/index.md:1]: TEST

Comment (DalM2023) [reply concepts/resilience-essentials/index.md:1]:
```TEST EDITED
---
```

Comment (cobusbernard) [reply concepts/resilience-essentials/index.md:1]: Thanks @DalM2023! @setheliot, please ignore these, we are testing access.

**Review by damonk10 — APPROVED** [review]

**Pull request merged**